quirks/password-rules.json

```diff
@@ -431,6 +431,9 @@
     "santander.de": {
         "password-rules": "minlength: 8; maxlength: 12; required: lower, upper; required: digit; allowed: [-!#$%&'()*,.:;=?^{}];"
     },
+    "secure.orclinic.com": {
+        "password-rules": "minlength: 6; maxlength: 15; required: lower, digit; allowed: ascii-printable;"
```

Comment:
Am I understanding the [documentation](https://developer.apple.com/documentation/security/password_autofill/customizing_password_autofill_rules) correctly that if `allowed` is not specified, it defaults to `ascii-printable`? If so, would it be better to omit `allowed: ascii-printable` from this rule?

Reply:
`allowed: ascii-printable` does indeed need to be included in this rule; otherwise, the only allowed character sets would be those that are `required` (`lower` + `digit`).

The `ascii-printable` default is only used if neither `required` nor `allowed` are specified.

Reply:
Thanks for the clarification! 😊